README.rst

```diff
@@ -36,16 +36,18 @@ Usage
     # ...and into which we install our project and its testing requirements.
     $ env/bin/pip install -e ".[testing]"
 
-3. Initialize the database.
+3. Initialize the database. Read all the options first, then choose according to your needs.
 
 .. code-block:: bash
 
-    # Use alembic to generate revisions before you run initialize.
+    # Use alembic to generate revisions before you run revision.
     $ env/bin/alembic -c development.ini revision --autogenerate -m "init"
     # Then you can use alembic to upgrade to that revision.
     $ env/bin/alembic -c development.ini upgrade head
-    # Or initialize the database.
-    $ env/bin/initialize_{{ cookiecutter.repo_name }_db development.ini
+    # Or initialize the database directly to latest revision.
+    $ env/bin/initialize_{{ cookiecutter.repo_name }}_db development.ini
+    # Later you can run database migrations.
+    $ env/bin/alembic -c development.ini ...
```

Comment:
Now I see the complete picture. Sorry I didn't fully understand on my first pass. I have a few more revisions, and then we're good to go. This is less verbose and clearer. The changes should be applied to the other two files as well for consistency.

```
3. Initialize the database.

.. code-block:: bash

    # First use alembic to generate revisions.
    $ env/bin/alembic -c development.ini revision --autogenerate -m "init"
    # Then you can use alembic to either:
    # (a) upgrade to that revision:
    $ env/bin/alembic -c development.ini upgrade head
    # (b) or initialize the database directly to the latest revision:
    $ env/bin/initialize_{{ cookiecutter.repo_name }}_db development.ini
    # Later you can run database migrations.
    $ env/bin/alembic -c development.ini ...
```